src/qforte/qubit_basis.h

```diff
@@ -40,8 +40,8 @@ class QubitBasis {
     /// set this state to zero
     void zero() { state_ = static_cast<basis_t>(0); }
 
-    /// return the address of the state
-    size_t add() const { return state_; }
+    /// Convenience function to return state_ as an index.
+    size_t index() const { return static_cast<size_t>(state_); }
```

Comment:
Because this is probably a new feature for you: `static_cast` is a way of saying very very explicitly that we're doing a type conversion.